pkg/issuer/acme/http/service.go

```diff
@@ -113,6 +112,15 @@ func buildService(crt *v1alpha1.Certificate, ch v1alpha1.ACMEOrderChallenge) *co
 			Selector: podLabels,
 		},
 	}
+
+	// http01 config is still optional, thus checking for presence and setting previous default
```

Comment:
technically it is not optional once this part of the code has been reached - however, we should keep this check anyway to avoid a nil dereference just in case 😄